### ISSUE TITLE
[Problem] Scale Type: Page for sections not stored upon file save

### ISSUE THREAD

**Issue by furgo16** [FreeCAD/FreeCAD#12001]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

When adding a section using the TechDraw workbench, if its `Base > Scale Type` property of that section is set to `Page` to use the containing page's scale, that setting is not saved. That is, whenever the file is reloaded, the setting will revert to `Custom` with `Scale` = 1. In the given example, this results in the section being scaled too big and off page.

![Captura de pantalla de 2024-01-15 23-39-57](https://github.com/FreeCAD/FreeCAD/assets/148809153/f70d46c5-7181-404c-9d7d-af5a864dd451)
1. The `Base > Scale Type` property of the section was initially set as `Page`. Then the file is saved and closed

![Captura de pantalla de 2024-01-15 23-39-26](https://github.com/FreeCAD/FreeCAD/assets/148809153/f478850f-3da3-481b-a29d-9f6d490953f4)
2. After reopening the file, the property is reverted to `Custom`. The section's scale is too big.

### Full version info

```shell
OS: Ubuntu Core 22 (ubuntu:GNOME/ubuntu-xorg)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.35580 (Git) Snap 896
Build type: Release
Branch: main
Hash: 2f710755bc5a1594e45a953b4d1082edd131fb65
Python 3.10.12, Qt 5.15.10, Coin 4.0.0, Vtk 7.1.1, OCC 7.7.1
Installed mods: 
  * BIM 2021.12.0
  * ExplodedAssembly
  * Assembly4 0.50.6
```

### Subproject(s) affected?

Techdraw

### Anything else?

[File to reproduce](https://github.com/FreeCAD/FreeCAD/files/13943566/Playpen.-.TechDraw.section.scale.bug.zip)

As a workaround, theSection's  `Base > Scale Type` property can be set to `Custom`, and then the `Base > Scale` property value be manually set to the correct value. Upon saving the file, that setting is remembered.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Roy-043** [comment]
Workaround 1: After changing the Scale Type to `Page`, change the Scale of the page to 0.11 and then back to 0.10.

Workround 2: Change the Scale Type to `Custom`, change the section scale to 0.10  and then change the Scale Type to `Page`.

**Comment by WandererFan** [comment]
The scale & type is being set wrong at creation.  It is using values from the DrawProjectionGoupItem (front) instead of the values from the parent DrawProjectionGroup.

**Comment by WandererFan** [comment]
Should be fixed for new documents by commits cf3189fef7 (simple section) and 0eee1c476a (complex section).

Existing documents with the issue will have to be corrected by hand by recreating the section view.
